### ISSUE TITLE
if i click on this mod nmm crashes

### ISSUE THREAD

**Issue by lazynick** [Nexus-Mods/Nexus-Mod-Manager#886]
as the title said if i download this mod and then try to click on it nmm hangs and eventually crashes
https://www.nexusmods.com/skyrimspecialedition/mods/17601

**Comment by jbostrus** [comment]
It's not the mod itself that's the problem. It's the content of the Nexus webpage that NMM is trying to figure out how to display on the side panel.

I'm not real good with RegEx stuff so this is more just to jump start another dev who might be able to solve the issue. The particular point of problem seems to be when the RegEx matcher tries to search for the urls links. In UI\Controls\HtmlLabel.cs ~ line 303 it does a match using LOOSE_URL_MATCHER. Some of the content on the mod page mentioned previously it doesn't like. I think I narrowed it down to a snippet within the Method 2 spoiler for the "How to use" section. There's actually quite a few occurrences of the sample type:

`[url=http://SKI_SettingsManagerInstance.SetOverride]SKI_SettingsManagerInstance.SetOverride[/url](\"ListLayout$defaults$label$textFormat$size\", \"12\")`

This causes the RegEx parser to freak out.

**Comment by lazynick** [comment]
yeah i thought something weird is happening because i repackaged the mod and added it manually and the problem went away
